Orange/widgets/tests/base.py

```diff
@@ -561,3 +566,61 @@ def _compare_selected_annotated_domains(self, selected, annotated):
         selected_vars = selected.domain.variables + selected.domain.metas
         annotated_vars = annotated.domain.variables + annotated.domain.metas
         self.assertLess(set(selected_vars), set(annotated_vars))
+
+
+class datasets:
+    @staticmethod
+    def path(filename):
+        dirname = os.path.join(os.path.dirname(__file__), "datasets")
+        return os.path.join(dirname, filename)
+
+    @classmethod
+    def data_1(cls):
```

Comment:
These methods could use more descriptive names. maybe missing_data_1? Telling what exactly is in the dataset would probably make them longer that 50 characters.